config/environments/test.rb

```diff
@@ -49,4 +49,8 @@
 
   # Use separate cache stores for parallel_tests
   config.cache_store = :file_store, Rails.root.join("tmp", "cache", "paralleltests#{ENV['TEST_ENV_NUMBER']}")
+
+  # Disables automatic creation of the test DB so that it has to be explicitly
+  # generated before running tests.
+  config.active_record.maintain_test_schema = false
```

Comment:
This change would require an additional step of running db:test:prepare, which is normally handled automatically under `rake test`.